### PR TITLE
(tlg4102)tlg4102.tlg001_tlg4102.tlg007

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated cts descriptions, fixed dates in metadata and added in page level links.

### CODE REVIEW

File: data/tlg4102/tlg001/__cts__.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4102" xml:lang="grc" urn="urn:cts:greekLit:tlg4102.tlg001">
   <ti:title xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr. 23)</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4102.tlg001.opp-grc1" workUrn="urn:cts:greekLit:tlg4102.tlg001">
-    <ti:label xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr.</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr. 23), 
-    Catenæ Græcorum Patrum in Novum Testamentum, Vol 1, Cramer, Oxford 1840</ti:description>
+    <ti:label xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr.)</ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 1.
```
Comment (lcerrato) [inline data/tlg4102/tlg001/__cts__.xml:5]: We don't generally use the dipthong (we would just transcribe to ae), but if you do, then I think you may need to have a Unicode declaration in the file.

File: data/tlg4102/tlg001/__cts__.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+      Cramer, John Anthony, editor. Oxford: Oxford Unviersity Pres, 1840.</ti:description>
```
Comment (lcerrato) [inline data/tlg4102/tlg001/__cts__.xml:6]: University misspelled

File: data/tlg4102/tlg003/__cts__.xml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-  <ti:title xml:lang="lat">Supplementum et varietas lectionis ad commentarium Chrysostomi in evangelium sancti Matthaei (e cod. Oxon. Bodl. Auct. T.1.4) </ti:title>
+  <ti:title xml:lang="lat">Supplementum et varietas lectionis ad commentarium Chrysostomi in evangelium sancti Matthaei (e cod. Oxon. Bodl. Auct. T.1.4)</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4102.tlg003.opp-grc1" workUrn="urn:cts:greekLit:tlg4102.tlg003">
-    <ti:label xml:lang="lat">Supplementum et varietas lectionis ad commentarium Chrysostomi in evangelium sancti Matthaei (e cod. Oxon. Bodl. Auct. T.1.4) </ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Supplementum et varietas lectionis ad commentarium Chrysostomi in evangelium 
-      sancti Matthaei (e cod. Oxon. Bodl. Auct. T.1.4), Cramer, Oxford, 1844 </ti:description>
+    <ti:label xml:lang="lat">Supplementum et varietas lectionis ad commentarium Chrysostomi in evangelium sancti Matthaei (e cod. Oxon. Bodl. Auct. T.1.4)</ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 1.
```
Comment (lcerrato) [inline data/tlg4102/tlg003/__cts__.xml:5]: Same two issues as above.

File: data/tlg4102/tlg004/__cts__.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <ti:title xml:lang="lat">Catena In Lucam (Typus B) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Misc. 182)</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4102.tlg004.opp-grc1" workUrn="urn:cts:greekLit:tlg4102.tlg004">
     <ti:label xml:lang="lat">Catena In Lucam (Typus B) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Misc. 182)</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Catena In Lucam (Typus B) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Misc. 182),
-    Cramer, Oxford, 1844</ti:description>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
+      Cramer, John Anthony, editor. Oxford: Oxford Unviersity Pres, 1841.</ti:description>
```
Comment (lcerrato) [inline data/tlg4102/tlg004/__cts__.xml:6]: same as above (æ versus ae) and "Unviersity"

File: data/tlg4102/tlg007/__cts__.xml [hunk 5]
```diff
@@ -1,8 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4102" xml:lang="grc" urn="urn:cts:greekLit:tlg4102.tlg007">
   <ti:title xml:lang="lat">Supplementum Et Varietas Lectionis Ad Catenam In Evangelium Sancti Joannis (e cod. Bodl. B.)</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4102.tlg007.opp-grc1" workUrn="urn:cts:greekLit:tlg4102.tlg007">
-    <ti:label xml:lang="lat">Supplementum Et Varietas Lectionis Ad Catenam In Evangelium Sancti Joannis (e cod. Bodl. B.)</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Supplementum Et Varietas Lectionis Ad Catenam In Evangelium Sancti Joannis (e cod. Bodl. B.),
-    Cramer, Oxford, 1844</ti:description>
+    <ti:label xml:lang="lat">Supplementum Et Varietas Lectionis Ad Catenam In Evangelium Sancti Joannis (e cod. Bodl. B.)
+    </ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
```
Comment (lcerrato) [inline data/tlg4102/tlg007/__cts__.xml:6]: same as other CTS files

File: data/tlg4102/tlg006/__cts__.xml [hunk 6]
```diff
@@ -2,7 +2,7 @@
   <ti:title xml:lang="lat">Supplementum et varietas lectionis ad catenam in evangelium sancti Lucae (e cod. Oxon. Bodl. Laud. 33)</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4102.tlg006.opp-grc1" workUrn="urn:cts:greekLit:tlg4102.tlg006">
     <ti:label xml:lang="lat">Supplementum et varietas lectionis ad catenam in evangelium sancti Lucae (e cod. Oxon. Bodl. Laud. 33)</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Supplementum et varietas lectionis ad catenam in evangelium sancti Lucae (e cod. Oxon. Bodl. Laud. 33),
-    Cramer, Oxford, 1844</ti:description>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
```
Comment (lcerrato) [inline data/tlg4102/tlg006/__cts__.xml:5]: same as other cts files

File: data/tlg4102/tlg005/__cts__.xml [hunk 7]
```diff
@@ -2,7 +2,7 @@
   <ti:title xml:lang="lat">Catena In Joannem (Catena Integra) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Auct. T.1.4)</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4102.tlg005.opp-grc1" workUrn="urn:cts:greekLit:tlg4102.tlg005">
     <ti:label xml:lang="lat">Catena In Joannem (Catena Integra) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Auct. T.1.4)</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Catena In Joannem (Catena Integra) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Auct. T.1.4),
-    Cramer, Oxford, 1844</ti:description>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
```
Comment (lcerrato) [inline data/tlg4102/tlg005/__cts__.xml:5]: same as other CTS files

**Review by lcerrato — CHANGES_REQUESTED** [review]

**Comment by lcerrato** [comment]
Hi @AlisonBabeu 
In all of these CTS files there are two minor issues:

1. "University" is misspelled
2. the diphthong is used instead of ae.  This is an editorial choice (I usually transcribe to two letters for simplicity) but it may be that if the dipthong is maintained that we want to be sure that a Unicode declaration is in every CTS file. This is inconsistent across all collections: ` <?xml version="1.0" encoding="UTF-8"?>`

So the typo needs changing but the dipthong can stay based on how you want this to appear. We should do a check on the Unicode declarations, though.

File: data/tlg4102/tlg001/__cts__.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg4102/tlg001/__cts__.xml:6]: Doh. Thanks. I will fix that as well as the changing the dipthong above!

**Comment by AlisonBabeu** [comment]
hey @lcerrato so I've changed the dipthongs to ae and fixed the spelling, and will not use the dipthong moving forward. Looking at the cts_.xml files for this textgroup question I don't see any Unicode declaration. In fact none of the cts_.xml files have the XML top level root element either or I am only supposed to find that in the TEI-XML files? Thanks!

**Review by lcerrato — APPROVED** [review]

**Comment by lcerrato** [comment]
@AlisonBabeu 
We have some in Perseus but I do not know if there was ever a standard practice established on that. Without it, it's very likely the special characters would break.
We could probably test this in the cts reading environment, but it's not an urgent issue.

**Pull request merged**